apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/KeyValuePairEnumerableExtensions.cs

```diff
@@ -23,7 +23,7 @@ namespace Google.Cloud.Diagnostics.Common
     /// Extension methods for converting KeyValuePair Enumerables to several types,
     /// including Protobuf well known types.
     /// </summary>
-    public static class KeyValuePairEnumerableExtensions
+    internal static class KeyValuePairEnumerableExtensions
```

Comment:
This isn't a breaking change because it was introduced since the last release, yes?

Reply:
Yes, that's just one from the previous PR.